website/pages/blog/postgres-cdc-to-any-destination.mdx

```diff
@@ -0,0 +1,98 @@
+
+## What is CDC?
+
+[CDC (Change-Data-Capture)](https://en.wikipedia.org/wiki/Change_data_capture) is a term used in a set of design patterns in databases where one can track changes to the data and act upon it.
+In PostgreSQL, CDC can be achieved via a feature called logical replication.
+
+## What is Logical Replication?
+
+PostgreSQL supports two types of replication: physical replication and logical replication.
+Physical replication creates a complete copy of the data, including the underlying storage structure,
+whereas logical replication only replicates changes to the data, such as insertions, updates, and deletions of rows in tables.
+Logical replication provides a more flexible and efficient way to replicate data as it only replicates changes to the data,
+rather than the entire dataset. This can help to minimize the overhead of replication and reduce the amount of disk space required to store the replicated data.
+
+## How Does Logical Replication Work?
+
+In PostgreSQL, logical replication is achieved by replicating changes made to the data in the form of write-ahead logs (`WALs`) records.
+These records contain all the necessary information to recreate the changes that were made to the data,
+including the type of change (insert, update, or delete),
+the time the change was made, and the actual data that was changed.
+
+The WAL records are written to disk on the master server,
+and a separate process, known as the WAL sender, sends the WAL records to the replica server.
+The replica server then applies the changes to its own copy of the data.
+This process happens in real-time, so the replica server's copy of the data is always up-to-date with the master server's copy.
+
+## How CloudQuery Uses Postgres Logical Replication to Sync Postgres to Any Other Destination
+
+CloudQuery PostgreSQL source plugin provides the ability to sync data and changes in real time from a PostgreSQL database, directly to any
+of the supported [destinations](/docs/plugins/destinations/overview), without the need for Kafka or any intermediary (You can still stream to Kafka or Google PubSub).
+This gives a much simpler setup to operate by just running one CloudQuery binary!
+
+In the following section, we will go through how our CloudQuery PostgreSQL source plugin works internally. (For running and configuring, refer to the [docs](https://www.cloudquery.io/docs/quickstart/macOS).)
+
+### Set `wal_level` to logical
+
+To use logical replication, you will need to make sure you started your database with `wal_level`.
+
+See our docs on how to set `wal_level` to logical in various environments.
+
+### Create a publication
+
+CloudQuery creates a publication for all or a subset of the specified tables via the [`CREATE PUBLICATION`](https://www.postgresql.org/docs/current/sql-createpublication.html) command.
```

Comment:
I don't feel good about linking to `QUOTED COMMANDS`. Maybe provide the links separately in parens, like so:
> `CREATE PUBLICATION` [docs](https://...)
Same goes in L63 and L79, L81